lib/smartsheet/webhook/webhook.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Smartsheet.Webhook do
+  @derive [Poison.Encoder]
+
+  defstruct [
+    :apiClientId,
+    :id,
+    :apiClientName,
+    :createdAt,
+    :disabledDetails,
+    :enabled,
+    :modifiedAt,
+    :sharedSecret,
+    :stats,
+    :status,
+    :callbackUrl,
+    :events,
+    :name,
+    :scope,
+    :scopeObjectId,
+    :subscope,
+    :version
+  ]
```

Comment:
you can `snake_case` these. via [recase](https://github.com/sobolevn/recase) we are processing the response body and always atomizing + snake_casing the strings https://github.com/fast-radius/smartsheet/blob/master/lib/smartsheet/http_client.ex#L74